openstates/sd/bills.py

```diff
@@ -159,51 +176,55 @@ def scrape_vote(self, bill, date, url):
         else:
             raise ScrapeError("Bad chamber: %s" % location)
 
-        committee = ' '.join(location.split(' ')[1:]).strip()
-        if not committee or committee.startswith('of Representatives'):
-            committee = None
+        # committee = ' '.join(location.split(' ')[1:]).strip()
```

Comment:
Can we still pass the committee to the vote via the `organization` field? cc @jamesturk